events/migrations/0001_initial.py

```diff
@@ -8,9 +7,9 @@
 import django.utils.timezone
 from django.conf import settings
 from django.db import migrations, models
-from django.utils.timezone import utc
 
 
+utc = datetime.timezone.utc
```

Comment:
Need to test on QA if this will break stuff (should be fine as functionality should be the same)